packages/xo-web/src/xo-app/sr/tab-xostor.js

```diff
@@ -40,7 +42,14 @@ const RESOURCE_COLUMNS = [
   },
   {
     name: _('vdi'),
-    itemRenderer: ({ vdiId }) => vdiId !== '' && <Vdi id={vdiId} />,
+    itemRenderer: ({ vdiId }) =>
+      vdiId !== '' && (
+        <Copiable data={vdiId}>
+          <Tooltip content={_('copyUuid', { uuid: vdiId })}>
+            <Vdi id={vdiId} />
+          </Tooltip>
+        </Copiable>
```

Comment:
Also, it looks like it's not usable like this because you can't reach the button with your mouse before it disappears. Or is it just me?